### DISCUSSION TITLE
limited access

### DISCUSSION THREAD

**Discussion by Razriel3** [cockpit-project/cockpit#17871] [General]
I am using cockpit in a YOCTO enviorment.
currently I am able to login with root permission and have all capabilies.
I have created another user in my system and gave him permissons to the following gruops:
> groups user
root adm cdrom sudo dip plugdev user

still when I login with this user I see on top message "limited access"
what do I need to do to have full capability for a user which is not root ?

**Comment by jelly** [comment]
If you click limited access does it ask you for your password and do you get "administrative access"?

**Comment by Razriel3** [comment]
If I click it I get a pop up window saying.
Something went wrong.
Spawn failed
